### DISCUSSION TITLE
getStr读取文件的问题

### DISCUSSION THREAD

**Discussion by TTTalen** [PFCCLab/PPOCRLabel#22] [Q&A] [answered]
我自己定义了一个，重新标注单张图片的函数。
在使用 action定义其名称时，使用
        AutoRecSingle = action(
            getStr("autoSingleRecognition"),
            self.autoSingleRecognition,
            "",
            "Auto",
            getStr("autoSingleRecognition"),
            enabled=False,
        )
时出现了错误，调试代码发现libs/stringBundle.py，读取resources/strings/strings-en.properties或是resources/strings/strings-zh-CN.properties时，并没有将我加入的autoSingleRecognition=Auto Single Recognition读写到字典中。具体如libs/stringBundle.py的第88行line = ustr(text.readLine())。并没有读取到我自己新增的行autoSingleRecognition=Auto Single Recognition。所以只能采取如下方法。
        AutoRecSingle = action(
            "单图片自动标注",
            self.autoSingleRecognition,
            "",
            "Auto",
            "单图片自动标注",
            enabled=False,
        )
请问下，具体是什么原因呢?

**Comment by GreatV** [comment, marked as answer]
是不是要重新生成一下资源
```
# Regenerate Resources
pyrcc5 -o libs/resources.py resources.qrc
```

**Reply by TTTalen** [reply]
好的，谢谢，是需要重新生成资源
